lti_consumer/migrations/0006_add_on_model_config_for_lti_1p1.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.17 on 2020-12-21 13:39
+
+from django.db import migrations, models
+import jsonfield.encoder
+import jsonfield.fields
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('lti_consumer', '0005_migrate_keyset_to_model'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='lticonfiguration',
+            name='config_id',
+            field=models.UUIDField(default=uuid.uuid4, editable=False, unique=True),
+        ),
+        migrations.AddField(
+            model_name='lticonfiguration',
+            name='lti_1p1_client_key',
+            field=models.CharField(blank=True, help_text='Client key provided by the LTI tool provider.', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='lticonfiguration',
+            name='lti_1p1_client_secret',
+            field=models.CharField(blank=True, help_text='Client secret provided by the LTI tool provider.', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='lticonfiguration',
+            name='lti_1p1_launch_url',
+            field=models.CharField(blank=True, help_text='The URL of the external tool that initiates the launch.', max_length=255),
+        ),
+        migrations.AddField(
+            model_name='lticonfiguration',
+            name='lti_config',
+            field=jsonfield.fields.JSONField(blank=True, default=dict, dump_kwargs={'cls': jsonfield.encoder.JSONEncoder, 'separators': (',', ':')}, help_text='LTI configuration data.', load_kwargs={}),
+        ),
+        migrations.AlterField(
+            model_name='lticonfiguration',
+            name='config_store',
+            field=models.CharField(choices=[('CONFIG_ON_XBLOCK', 'Configuration Stored on XBlock fields'), ('CONFIG_ON_DB', 'Configuration Stored on this model')], default='CONFIG_ON_XBLOCK', max_length=255),
```

Comment:
Probably not a big deal, but I wonder if the default should now be `CONFIG_ON_DB`..

Reply:
The config on db path isn't fully developed, and doesn't support LTI1.3, so until all the paths for that are supported I think we should default to config on XBlock.